tests/adderall_test.hy

```diff
@@ -29,69 +30,95 @@
   None)
 
 (defn test-cons []
-  (assert-all-equal (cons 'a None)
-                    (cons 'a [])
-                    (cons 'a (,))
-                    (cons 'a '())
-                    ['a])
-  (assert-all-equal (cons None 'a)
-                    (cons [] 'a)
-                    (cons (,) 'a)
-                    (cons '() 'a))
-  (assert-not-equal (cons 'a None)
-                    (, 'a))
-  (assert-equal (cons 'a None)
-                (list (, 'a)))
+  (assert-all-equal
```

Comment:
All of the `cons` tests that don't explicitly use `HyExpression`s should be removed; they're covered by the tests in `python-cons`.